plugins/database/redshift/redshift_test.go

```diff
@@ -48,38 +47,40 @@ var (
 	vaultACC = "VAULT_ACC"
 )
 
-func redshiftEnv() (url string, user string, password string, errEmpty error) {
+func interpolateConnectionURL(url, user, password string) string {
+	return fmt.Sprintf("postgres://%s:%s@%s", user, password, url)
+}
+
+func redshiftEnv() (connURL string, url string, user string, password string, errEmpty error) {
 	errEmpty = errors.New("err: empty but required env value")
```

Comment:
Ew. This doesn't tell the user what env variables they are missing.

Reply:
Fixed.